atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -240,6 +240,8 @@ private static SerializableTransactionManager create(
         for (Schema schema : allSchemas) {
             Schemas.createTablesAndIndexes(schema, kvs);
         }
+        // Prime the key value service with logging information.
+        kvs.rehydrateLoggingArgSupplier();
```

Comment:
maybe add a note that we'll need to change this if we support dynamic table creation